server/src/scripts/TextDefinitionsParser.js

```diff
@@ -6,80 +6,93 @@ const fs = require("fs");
 const Utils = require("../Utils");
 
 const workbook = XLSX.readFile("Dungeonz.io translations.xlsx");
-
-const firstSheetName = workbook.SheetNames[0];
-// Get the worksheet.
-const worksheet = workbook.Sheets[firstSheetName];
-// Some characters to loop though, to go across the columns.
-const chars = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N"];
-const defsAsJSON = {};
-const idNameCol = "A";
-// The row that the language names are on.
-const languageNameRow = 8;
-// The row that the first definition is on for each column.
-const firstDefRow = 9;
-// The current row being accessed.
-let currentDefRow = 9;
-// The index of the column character in the chars array for the current language. Incremented as the languages are gone through.
-let currentLanguageCharIndex = 1;
-// The value of that index in the chars array.
-let currentLanguageCol = chars[currentLanguageCharIndex];
-// The name of the language i.e. 'English', 'French'.
-let currentLanguageName = worksheet[currentLanguageCol + languageNameRow].v;
-// An object of the text defs that have been added to the current language so far.
-let currentLanguageDefs;
-
-// Add an object for the current language.
-defsAsJSON[currentLanguageName] = {};
-
-while (true) {
-    // Check if this language is valid. The previous one might have been the last one, in which case end.
-    if (currentLanguageName === undefined) break;
-
-    // Shorthand.
-    currentLanguageDefs = defsAsJSON[currentLanguageName];
-
-    // Add the ID name as a key. Null by default, value added if found.
-    currentLanguageDefs[worksheet[idNameCol + currentDefRow].v] = null;
-
-    // Check there is a value at this row for the current language.
-    if (worksheet[currentLanguageCol + currentDefRow] !== undefined) {
-        // Add the value.
-        currentLanguageDefs[
-            worksheet[idNameCol + currentDefRow].v
-        ] = worksheet[currentLanguageCol + currentDefRow].v;
+const firstTabIndex = 2;
```

Comment:
The old sheets have been removed from the translations spreadsheet, so this is not needed any more.